packages/EasyCore/src/Doctrine/Events/EntityUpdatedEvent.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Doctrine\Events;
+
+use EonX\EasyCore\Interfaces\DatabaseEntityInterface;
+use Symfony\Contracts\EventDispatcher\Event;
```

Comment:
Do we really need to extend the symfony event class?
The idea of this is to work with doctrine regardless of the framework it is used with.

Reply:
Not needed. Removed.